pyproject.toml

```diff
@@ -5,3 +5,12 @@ exclude= '''
   | tests/kitchen
   | templates
 )/
+
+
+[tool.isort]
```

Comment:
This actually makes isort and black fight over imports.
Let's leave the old `.isort.cfg` as it was?

Reply:
I should have added the `ensure_newline_before_comments` flag, but I couldn't get it to work correctly :suspect: 

I'm down with switching back, though when there's a version 5 release of isort that has mo' bettah support, I would prefer heading back to pyproject.toml

Also FWIW when toml hits the version 1 release, Python will be including a toml parser in the stdlib :tada:

Reply:
Cool, we'll switch when it lands :)

Reply:
Okay, turns out I figured out how to fix it. My bad :upside_down_face: